packages/airnode-validator/src/config/config.ts

```diff
@@ -173,7 +173,7 @@ export const apiKeySchema = z.string().min(30).max(120);
 
 export const corsOriginsSchema = z.array(z.string());
 
-export const enabledGatewaySchema = z
+export const EnabledGateway = z
```

Comment:
Sorry, I've made an incorrect find/replace when renaming the type. Will fix.

I'm also looking into the unit and e2e tests, because I can't seem to get them passing for this PR.